server/src/controllers/common.es6

```diff
@@ -6,6 +6,50 @@ const TempOutfitterApplication = require('../models/tempoutfitter-application.es
 const commonControllers = {};
 
 commonControllers.getAllOpenApplications = (req, res) => {
+  const statusGroup = req.params.statusGroup;
+  let orCondition = [];
+  switch (statusGroup) {
+    case 'pending':
+      orCondition = [
+        {
+          status: 'Received'
+        },
+        {
+          status: 'Hold'
+        }
+      ];
+      break;
+    case 'accepted':
+      orCondition = [
+        {
+          status: 'Accepted'
+        }
+      ];
+      break;
+    case 'rejected':
+      orCondition = [
+        {
+          status: 'Returned'
+        }
+      ];
+      break;
+    case 'cancelled':
+      orCondition = [
+        {
+          status: 'Cancelled'
+        }
+      ];
+      break;
+    case 'expired':
+      orCondition = [
+        {
+          status: 'Expired'
+        }
+      ];
+      break;
+    default:
+      res.status(404).send();
+  }
```

Comment:
This is a style thing, but it would probably reduce the amount of code and make it less error prone if this were a map of orConditions with status as keys.